content/docs/how-to-contribute.md

```diff
@@ -9,170 +9,197 @@ redirect_from:
 
-1. Fork [the repository](https://github.com/facebook/react) and create your branch from `master`.
-2. Run `yarn` in the repository root.
-3. If you've fixed a bug or added code that should be tested, add tests!
-4. Ensure the test suite passes (`yarn test`). Tip: `yarn test --watch TestName` is helpful in development.
-5. Run `yarn test-prod` to test in the production environment. It supports the same options as `yarn test`.
-6. If you need a debugger, run `yarn debug-test --watch TestName`, open `chrome://inspect`, and press "Inspect".
-7. Format your code with [prettier](https://github.com/prettier/prettier) (`yarn prettier`).
-8. Make sure your code lints (`yarn lint`). Tip: `yarn linc` to only check changed files.
-9. Run the [Flow](https://flowtype.org/) typechecks (`yarn flow`).
-10. If you haven't already, complete the CLA.
+1. Forkez [le dépôt](https://github.com/facebook/react) et créez votre branche depuis `master`.
+2. Lancez `yarn`  à la racine du dépôt.
+3. Si vous avez réparé un bug ou ajouté  du code qui devrait être testé, ajouter des tests !
+4. Assurez-vous que les suites de test passent (`yarn test`). Astuce : `yarn test --watch NomDuTest` est très utile en phase de développement.
+5. Lancez `yarn test-prod` pour tester dans l'environnement de production. Cette commande accepte les même options que `yarn test`.
+6. Si vous avez besoin d'un debugger, lancez `yarn debug-test --watch NomDuTest`, ouvrez `chrome://inspect`, et appuyez sur “Inspecter”.
+7. Formattez votre code avec [prettier](https://github.com/prettier/prettier) (`yarn prettier`).
+8. Assurez-vous que votre code passe la vérification du linter (`yarn lint`). Astuce : `yarn linc` pour ne vérifier que les fichiers qui ont été changés.
+9. Lancez les vérifications de types [Flow](https://flowtype.org/) (`yarn flow`).
+10. Si vous ne l'avez pas encore fait, complétez le CLA.
 
-### Contributor License Agreement (CLA) {#contributor-license-agreement-cla}
+***************
 
-In order to accept your pull request, we need you to submit a CLA. You only need to do this once, so if you've done this for another Facebook open source project, you're good to go. If you are submitting a pull request for the first time, just let us know that you have completed the CLA and we can cross-check with your GitHub username.
+### Accord de license de contributeur (NdT Contributor License Agreement) (CLA) {#contributor-license-agreement-cla}
 
-**[Complete your CLA here.](https://code.facebook.com/cla)**
 
-### Contribution Prerequisites {#contribution-prerequisites}
+Pour que nous puissions accepter votre pull request, il est nécessaire que vous soumettez un CLA. Vous ne devez faire cela qu'une seule fois, donc si vous l'avez déjà fait pour un autre projet open source de Facebook, pas besoin de le refaire. Si vous soumettez une pull request pour la première fois, dites-nous simplement que vous avez déjà complété le CLA et nous pourrons vérifier cela avec votre nom d'utilisateur GitHub.
```

Comment:
In order to accept your pull request, we need you to submit a CLA. You only need to do this once, so if you've done this for another Facebook open source project, you're good to go. If you are submitting a pull request for the first time, just let us know that you have completed the CLA and we can cross-check with your GitHub username.

```suggestion
Afin que nous puissions accepter votre _pull request_, nous avons besoin que vous soumettiez un CLA *(Contributor License Agreement, NdT)*. Vous n’avez besoin de faire ça qu'une seule fois, donc si vous l'avez déjà fait pour un autre projet open source de Facebook, tout va bien. Si vous envoyez une _pull request_ pour la première fois, dites-nous simplement que vous avez déjà rempli le CLA et nous pourrons le vérifier sur base de votre identifiant GitHub.
```